communication.py

```diff
@@ -187,7 +197,7 @@ def pckParser(self, line):
         self.widget.graphs.update_sp_tmp_v(abs(speed), tempe, b_voltage)
```

Comment:
all lines below here should be in comment. this part is unusable for now. you should check that is your parsing valid. visualize data later.